### PR TITLE
feat: add coin type

### PR DESCRIPTION
**Pull request by Tuditi**
# Description of change

Added coinType enum  to createAccountPayload

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Ran the examples

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/examples/1-create-account.js [hunk 1]
```diff
@@ -3,42 +3,17 @@
...
-
-    initLogger({
-        color_enabled: true,
-        outputs: [{
-            name: './wallet.log',
-            level_filter: 'debug'
-        }]
-    })
-    
-    const manager = new AccountManager({
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/1-create-account.js:19]: Why is that moved into another file?

Comment (Tuditi) [reply bindings/nodejs/examples/1-create-account.js:19]: Because it was duplicated in every example. I thought we could just enable it by default

Comment (Thoralf-M) [reply bindings/nodejs/examples/1-create-account.js:19]: Will approve now, but soon it will not be necessary anymore to add all the fields every time and then it shouldn't be moved to another file I think

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]
